include/dsVideoPort.h

```diff
@@ -390,7 +394,9 @@ dsError_t  dsIsDTCPEnabled (intptr_t handle, bool* pContentProtected);
  /**
  * @brief Indicates whether a video port is HDCP protected.
  * 
- * This function indicates whether the specified video port is configured for HDCP
+ * For sink devices, this function returns the authentication status as dsHDCP_STATUS_AUTHENTICATED 
```

Comment:
dsIsHDCPEnabled  - This function does not returns - dsHDCP_STATUS_AUTHENTICATED. It gives the HDCP protection status. Rephrase the sentence as below
"For sink devcies, this function always gets a connection status of "true" because the sink display is internal(dsVIDEOPORT_TYPE_INTERNAL)"

Reply:
Updated